test/README.md

```diff
@@ -27,6 +27,30 @@ Useful if you built your own image into your repository.
 
 `../hack/run-tests.sh --concurrent {NB_OF_CONCURRENT_TESTS}`
 
+### Run Kogito Runtime tests
+
+The Kogito Runtime tests have been disabled because it cannot be integrated to either the CI pipelines yet. 
+In order to run these tests locally, it requires:
+- an Openshift cluster
```

Comment:
```suggestion
- an Openshift or Kubernetes cluster
```
:)

Reply:
I will add this change. However, note that the tests are not ready for Kubernetes yet. This support will come as part of another ticket.